components/universal-widget/package.json

```diff
@@ -24,6 +29,7 @@
     "babel-loader": "^7.1.1",
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
```

Comment:
Where do we have used stage-2 syntax?

Reply:
babel-preset-es2015 doesn't support the rest spread operator. babel-preset-stage-2 is used to facilitate that.